erehwon/erehwon/templates/registration/password_reset_form.html

```diff
@@ -3,11 +3,12 @@
 {% load static %}
 
 {% block layout %}
+
     <div class="passwordreset">
 
-        <div class="passwordreset-header">
-            <a href="#"> < back to Erehwon </a>
-        </div>
+        <header>
+            <a href="/accounts/login"> < back to Erehwon </a>
```

Comment:
i think this should be `&lt;` instead of`<` back to Erehwon  .. and we may have to change this to other templates too